### PR TITLE
Adds highlights to projects response

### PR DESCRIPTION
**Pull request by alianam**
### What

Adds `highlights` to projects response.

### SDK Release Checklist

- [ ] Have you added an integration test for the changes?
- [ ] Have you built the package locally and made queries against it successfully?
- [ ] Did you update the changelog?
- [ ] Did you bump the package version?
- [ ] For breaking changes, did you plan for the release of the new SDK versions and deploy the API to production?

### CODE REVIEW

File: test/integration/projects.test.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ describe('Project Integration', function () {
     expect(projectResponse.data.mechanism).to.be.a('string');
     expect(projectResponse.data.latitude).to.be.a('number');
     expect(projectResponse.data.longitude).to.be.a('number');
+    expect(projectResponse.data.highlights).to.be.a('array');
```
Comment (kleinjm) [inline test/integration/projects.test.js:23]: [nit] does 
```suggestion
    expect(projectResponse.data.highlights).to.be.an('array');
```
work?

**Review by kleinjm — CHANGES_REQUESTED** [review]
Let's see if you can fix the diff on the `package-lock.json`. Otherwise, this LGTM

File: test/integration/projects.test.js (hunk 1 above)
Comment (alianam) [reply test/integration/projects.test.js]: Yes it does!

**Review by pcothenet — APPROVED** [review]
Nice!

**Review by kleinjm — APPROVED** [review]

**Review by kleinjm — APPROVED** [review]
LGTM. Failing test doesn't seem related

**Review by pcothenet — APPROVED** [review]

**Pull request merged**